extras/rya.streams/query-manager/src/test/java/org/apache/rya/streams/querymanager/QueryManagerIT.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.rya.streams.querymanager;
+
+import org.junit.Before;
+
+public class QueryManagerIT {
+    @Before
+    public void setup() {
+        //
+    }
+
+    // Test No query change log sources, add a new one
+
+    // Test existing query change log sources, add a new one with same rya instance
+    
+    // Test No query change log sources, remove one
```

Comment:
This case is only in the constructor.

Reply:
I should change this to say QueryChangeLogSource with no QueryChangeLogs and a notification with a remove gets triggered